### ISSUE TITLE
Corrupt output in travis with default progress logger

### ISSUE THREAD

**Issue by dantleech** [phpbench/phpbench#233]
The default report manipulates the cursor, which breaks travis output. Maybe this can be avoided by determining the capabilities of the terminal, or perhaps the default progress logger should not manipulate the cursor.

````
\PHPCR\Benchmark\InsertBench

s	μSD/r 0.007ms	μRSD/r: 0.18%

s	μSD/r 0.335ms	μRSD/r: 1.86%

s	μSD/r 23.281ms	μRSD/r: 11.29%

s	μSD/r 379.562ms	μRSD/r: 7.67%

\PHPCR\Benchmark\Query\QueryBench

s	μSD/r 0.066ms	μRSD/r: 0.86%

s	μSD/r 0.112ms	μRSD/r: 1.36%

s	μSD/r 0.104ms	μRSD/r: 1.26%

s	μSD/r 0.122ms	μRSD/r: 1.52%

s	μSD/r 0.033ms	μRSD/r: 0.43%

s	μSD/r 0.024ms	μRSD/r: 0.30%

s	μSD/r 0.042ms	μRSD/r: 0.53%
````

and

````
\PhpBench\Tests\Functional\benchmarks\ArrayKeysBench

\CostOfCalling

s        R4 I0 P0 s        R4 I1 P0 s        R4 I2 P0 s        R4 I3 P0 s        R4 I4 P0 s        R4 I5 P0 s        R4 I6 P0 s        R4 I7 P0 s        R4 I8 P0 s        R4 I9 P0 	μ/r: 4.137μs	μSD/r 0.194μs	μRSD/r: 4.68%

s           R4 I0 P0 s           R4 I1 P0 s           R4 I2 P0 s           R4 I3 P0 s           R4 I4 P0 s           R4 I5 P0 s           R4 I6 P0 s           R4 I7 P0 s           R4 I8 P0 s           R4 I9 P0 	μ/r: 4.530μs	μSD/r 0.146μs	μRSD/r: 3.23%

s    R4 I0 P0 s    R4 I1 P0 s    R4 I2 P0 s    R4 I3 P0 s    R4 I4 P0 s    R4 I5 P0 s    R4 I6 P0 s    R4 I7 P0 s    R4 I8 P0 
````

**Comment by dantleech** [comment]
Could be addressed by disabling control characters when the `CONTINUOUS_INTEGRATION` environment variable is set: https://github.com/travis-ci/travis-ci/issues/1337

**Comment by dantleech** [comment]
Fixed in #235
